doc/manual/src/contributing/experimental-flags.md

```diff
@@ -0,0 +1,68 @@
+This section describes the notion of *experimental features*, and how it fits into the big picture of the development of Nix.
+
+# What are experimental features?
+
+[Since Nix 2.4](../release-notes/rl-2.4.md), some features of Nix can be flagged as experimental.
+Experimental features can be changed or removed at any time, and users must [explicitly enable them].
+This allows accessing new functionality that is not yet stable without unwittingly relying on them.
+
+[explicitly enable them]: ../nix-conf.md#conf-expeirmental-features
```

Comment:
```suggestion
[explicitly enable them]: ../nix-conf.md#conf-experimental-features
```

Reply:
BTW I'm really opposed to the "footnote" style of linking. It requires duplicating text like "explicitly enable them", meaning that if you change that text, you have to remember to update it in two places and not make a typo. It's much less verbose and error-prone to just write:

> ... and users must [explicitly enable them](../nix-conf.md#conf-experimental-features).